docs/functions.rst

```diff
@@ -118,7 +118,7 @@ This allows you to run a new Pod from a Blueprint.
    :align: left
    :widths: 5,5,5,15
 
-   `namespace`, Yes, `string`, namespace in which to execute
+   `namespace`, No, `string`, namespace in which to execute (if not mentioned will create pod under ``kanister`` namespace)
```

Comment:
Let's also mention where the pod will be created if we don't specify the namespace.

Reply:
Wrote `(if not mentioned will create pod under ``kanister`` namespace)` in the arg description. Do we need to add more context or update this line?

Reply:
aah, I missed that I think. Let's use controller namespace instead of kanister namespace.

Reply:
Cool. I'll make this update. Thanks. 👍🏼